syft/format-options/options.go

```diff
@@ -0,0 +1,6 @@
+package options
+
+// Format has options to be used by SBOM formats
+type Format struct {
```

Comment:
I see why this is here, however, it is a little awkward (it's only purpose is for the state of template and isn't clear how to use this for more formats in the future). Is there a better way to do this?

Reply:
Right, I removed it. Now the template output receives the param directly.